src/runtime/virtcontainers/pkg/resourcecontrol/controller.go

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2021 Apple Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package cgroups
```

Comment:
should the package name be updated as well?

Reply:
Yes :) Fixed as well.